keps/sig-api-machinery/20190927-insecure-backend-proxy.md

```diff
@@ -220,3 +221,60 @@ Similar to the `Drawbacks` section the `Alternatives` section is used to highlig
 Use this section if you need things from the project/SIG.
 Examples include a new subproject, repos requested, github details.
 Listing these here allows a SIG to get the process for these resources started right away.
+
+## Production Readiness
+
+  * Feature enablement and rollback
+    - How can this feature be enabled / disabled in a live cluster?
+      Once GA, this feature cannot be disabled at a cluster-wide level. It is the decision of the client whether or not
```

Comment:
What about pre-GA? Will there be a feature gate?
[Would the form from kubernetes/community#4307 be more clear?]

Reply:
I think we should focus much more on Alpha/Beta, not just GA (I think we pretty much always enable the feature for the first time when in Beta).

Reply:
yeah, we need to make clear which of these questions apply to each phase. disablement doesn't apply to any non-optional GA features